pages/linux/dmidecode.md

```diff
@@ -0,0 +1,23 @@
+# dmidecode
+
+> Display the DMI (some say SMBIOS) table contents in a human-readable format as root user.
+
+- Show all DMI table contents:
+
+`sudo dmidecode`
+
+- Show the BIOS version:
+
+`sudo dmidecode -s bios-version`
+
+- Show the system serial number:
+
+`sudo dmidecode system-serial-number`
+
+- Show the whole BIOS information:
```

Comment:
Perhaps `Show the BIOS information` would read better here?

Reply:
Actually this command shows all the BIOS information, 
I'm not sure if I should use `Show the all BIOS information` or just `Show the BIOS information.`

Reply:
or `Show all the BIOS information`